perf/local_lat.js

```diff
@@ -19,7 +19,7 @@ rc = s.bind(bind_to);
 assert(rc >= 0);
 
 var i = 0;
-s.on('message', function (data) {
+s.on('data', function (data) {
```

Comment:
It looks like we'll be introducing breaking changes with this then, since users of this lib now have to listen for on data rather than on message?